tests/vmi_multus_test.go

```diff
@@ -1388,31 +1424,29 @@ func validatePodKubevirtResourceName(virtClient kubecli.KubevirtClient, vmiPod *
 	return nil
 }
 
-func defaultCloudInitNetworkData() string {
-	networkData, err := libnet.CreateDefaultCloudInitNetworkData()
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create default cloud init network data for SRIOV")
-	return networkData
-}
-
-func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress, ipAddress string) string {
+func cloudInitNetworkDataWithStaticIPsByMac(nicName, macAddress, ipAddress string) (string, error) {
 	networkData, err := libnet.NewNetworkData(
 		libnet.WithEthernet(nicName,
 			libnet.WithAddresses(ipAddress),
 			libnet.WithNameserverFromCluster(),
 			libnet.WithMatchingMAC(macAddress),
 		),
 	)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create static IPs by mac address cloud init network data")
-	return networkData
+	if err != nil {
+		return "", err
+	}
+	return networkData, nil
 }
 
-func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) string {
+func cloudInitNetworkDataWithStaticIPsByDevice(deviceName, ipAddress string) (string, error) {
 	networkData, err := libnet.NewNetworkData(
 		libnet.WithEthernet(deviceName,
 			libnet.WithAddresses(ipAddress),
 			libnet.WithNameserverFromCluster(),
 		),
 	)
-	ExpectWithOffset(1, err).ToNot(HaveOccurred(), "should successfully create static IPs by device name cloud init network data")
-	return networkData
+	if err != nil {
+		return "", err
```

Comment:
Same, is the info enough to understand where we failed?

Reply:
same here